ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2494,13 +2494,15 @@ def test073_create_new_existing_order_with_deleted_order_number(self):
         self.info('asserting the order with order number {} is created'.format(order_no))
         self.assertIn(order_no_with_year, results[0].text.replace("'", ""))
 
-    @skip("https://modeso.atlassian.net/browse/LIMSA-299")
+    #@skip("https://modeso.atlassian.net/browse/LIMSA-299")
     def test074_create_existing_order_change_contact(self):
         """
          Create existing order then change the contact for this existing one,
          all old records with the same order number will update its contact.
 
          LIMS-4293
+
+         LIMS-5818 - added departments assertion
         """
         res, payload = self.orders_api.create_new_order()
```

Comment:
replace it with :
`        res, payload = self.orders_api.create_order_with_department()
`

to make sure that departments updated, you need to have department first

Reply:
done